src/views/Create.vue

```diff
@@ -195,7 +198,9 @@ export default {
       start: '',
       end: '',
       snapshot: '',
-      metadata: {},
+      metadata: {
+        plugins: {}
```

Comment:
It's better if we dont add a "plugins" object by default here, if you want to check if the plugin setup if valid you can use optional chaining like this:
```
const isSafeSnapPluginValid = form.value.metadata?.plugins?.safeSnap
 ? form.value.metadata.plugins.safeSnap.valid : true;
```

Reply:
It's there so `form.metadata.plugins.safeSnap` can be set as the model to the `PluginSafeSnapConfig` on mount, as now the transactions block is always present on a new proposal.
We could also remove the `plugins` object and create a property for the SafeSnap plugin.